addons/crm/tests/test_crm_lead_multicompany.py

```diff
@@ -101,8 +101,7 @@ def test_lead_mc_company_computation_env_user_restrict(self):
         lead = LeadUnsyncCids.sudo().create({
             'name': 'My Lead MC',
         })
-        self.assertFalse(lead.company_id,
```

Comment:
Well this test was added on purpose. This should not change as this is required: we had crashes in assignment process due to company management and we should not get them back.

Reply:
Why shouldn't the company be set?